xformers/benchmarks/LRA/code/config.json

```diff
@@ -80,7 +80,7 @@
             "eval_frequency": 50,
             "num_train_steps": 10000,
             "num_eval_steps": 62,
-            "gradient_accumulation": 1
+            "gradient_accumulation": 2
```

Comment:
compositional takes more memory, so this task does not pass on a 8GB gpu (desktop 3080) without bumping up the accumulation